cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -754,38 +754,148 @@
 
   <property>
     <name>kafka.default.replication.factor</name>
+    <value>${kafka.server.default.replication.factor}</value>
+    <description>
+      CDAP Kafka service replication factor (deprecated: replaced with kafka.server.replication.factor)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.dirs</name>
```

Comment:
and if that's the case, the old one is called `kafka.log.dir` (without `s` at the end).